t/ui/01-list.t

```diff
@@ -71,8 +71,8 @@ sub schema_hook {
             DISTRI    => 'opensuse',
             FLAVOR    => 'NET',
             MACHINE   => '64bit',
-            VERSION   => '13.1'
-        });
+            VERSION   => '13.1',
+            settings  => [{key => 'JOB_TEMPLATE_NAME', value => 'kde_test'}]});
```

Comment:
better avoid the suffix "test", it's very ambiguous in this context. How about "kde_variant1"?